internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -46,19 +49,18 @@ const (
 	Tooling Component = "tooling"
 )
 
-type ControlPlaneComponentsVersion struct {
-	HyperkubeVersion string
-	EtcdVersion      string
+type ControlPlaneHostVersion struct {
```

Comment:
I'm not sure about this naming, this is not only about control planes, but also about worker nodes, right?

Reply:
@ereslibre Thanks for pointing out. Change to `ComponentHostVersion` and `ComponentContainerVersion`. Hope this naming is better than before :)